README.md

```diff
@@ -82,9 +82,9 @@ multiple departments take part in the festivals to showcase their skills, produc
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [Live Demo Link](https://codewithhyder.github.io/gurez-festival/)
 
-- [Watch Presentation]()
+- [Watch Presentation](https://www.loom.com/share/954565bd249a4b27b918eedb441eba61)
```

Comment:
- [ ] Thank you for your presentation. According to the project requirement, I recommend you to record another video including:
- A description of the project.
- A demo of the project features (different pages and different screen sizes). Here you should navigate to this pages and show all required features.
- You should also highlight some interesting piece of code or something you built that you are very proud of. (  Show these parts in VS code)